lib/src/rendering/overflow_view.dart

```diff
@@ -196,9 +195,9 @@ class RenderOverflowView extends RenderBox
   }
 
   void performFlexibleLayout() {
-    RenderBox child = firstChild;
+    RenderBox? child = firstChild;
```

Comment:
If there is no first child we can set a default size and return early instead;, and then child would not be able to be null